docs/reference/modules/_template.md

```diff
@@ -1,9 +1,15 @@
+---
+extends: '@apostrophecms/module'
```

Comment:
I'm using frontmatter since it seems likely that we'll want to have more programmatic access to this at some point.